examples/with-nestjs/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "with-typescript",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "node --inspect index.js",
+    "build": "next build",
+    "start": "NODE_ENV=production node index.js",
+    "type-check": "tsc"
+  },
+  "dependencies": {
```

Comment:
There's a lot of dependencies here. Generally we recommend splitting your frontend and backend, as React SSR wouldn't impact the performance of the API in that case.

Reply:
Hi, I cleanup some unused dependencies. 
I agree about recommendation of split frontend and backend, but sometimes we need consume services with CORS issues. Or simple make some transformation on the response of webservices. For that situations I think this solution.

Reply:
In that case I think this should be a separate repository 🕵️

Reply:
I think this cold be useful to Next.js users. The [pass-server-data](https://github.com/zeit/next.js/tree/canary/examples/pass-server-data) example do same thing using Express directly. But use Nest.js for these situations gives more flexibility to developers.

Reply:
Ideally the example should only show rendering, as APIs should be external. I'm actually not a fan of pass-server-data and might remove it soon as it shows an anti-pattern.